lib/manageiq/style/cli.rb

```diff
@@ -15,16 +15,15 @@ def parse_cli_options
         require 'optimist'
 
         Optimist.options do
-          usage    "manageiq-version [OPTIONS]"
-          synopsis "The ManageIQ communities style configuration utility"
-          version  "v#{Manageiq::Style::VERSION}\n"
+          usage    "[OPTIONS]"
+          synopsis "\nThe ManageIQ community's style configuration utility."
+          version  "v#{ManageIQ::Style::VERSION}\n"
 
-          opt :install, "Install or update the style configurations", :default => false
+          opt :install, "Install or update the style configurations", :default => false, :required => true
         end
       end
 
       def run
-        version if @opts[:version]
```

Comment:
This never could have executed because version is handled by optimist itself, and besides, there's no `def version` here.